elements.go

```diff
@@ -30,3 +30,24 @@ type groupMembership struct {
 	XMLName xml.Name        `xml:"DAV: group-membership"`
 	Hrefs   []internal.Href `xml:"href"`
 }
+
+// ConditionalHeaderValue represents the value of a conditional header
+// according to RFC 2068 section 14.25 and RFC 2068 section 14.26
+// The (optional) value can either be a wildcard or an ETag.
+type ConditionalHeaderValue string
```

Comment:
Since there are other conditional header fields (e.g. If-Modified-Since), maybe we can find another name? `ETagMatch`? `ConditionalMatch`?

Reply:
Changed to `ConditionalMatch`, thanks!